cmds/exp/hdparm/hdparm.go

```diff
@@ -13,46 +13,51 @@
 //
 // We also have no plans to support ata12. It's 2019.
 //
+// Unlike the standard hdparm command, we do not allow empty
+// arguments for commands requiring a password.
+//
 // Synopsis:
-//     hdparm [--security-unlock] [--user-master|--timeout] [device ...]
+//     hdparm [--identify] [--security-unlock] [--user-master|--timeout] [device ...]
+//
 package main
 
 import (
 	"flag"
 	"fmt"
 	"log"
-	"os"
 
 	"github.com/u-root/u-root/pkg/scuzz"
 )
 
 // Because all the verbs and options are switches, hence global,
 // we can have all the functions take one Disk as a parameter and
-// return an error. This simplifies other aspects of this program.
+// return a Response and error. This simplifies other aspects of this program.
 type op func(scuzz.Disk) error
 
 var (
-	verbose = flag.Bool("v", false, "verbose log")
-	debug   = func(string, ...interface{}) {}
-	unlock  = flag.String("security-unlock", "", "Unlock the drive")
-	master  = flag.Bool("user-master", false, "Unlock master (true) or user (false)")
-	timeout = flag.Uint("timeout", 15000, "Timeout for operations")
-	verbs   = map[string]op{
-		"--security-unlock": unlockop,
-	}
+	verbose  = flag.Bool("v", false, "verbose log")
+	debug    = func(string, ...interface{}) {}
+	unlock   = flag.String("security-unlock", "", "Unlock the drive")
```

Comment:
why is that a string and not a bool? I completely missed that lol

Reply:
yeah wonderful huh? the switches are a mix of strings and bools and who knows what. Why I love the command so much.

Reply:
oh and on the main page, to set an empty password, you say NULL

forget that.

Reply:
oh it's the password. we should probably support a password? we probably need it?

Reply:
Oh, you do support it. Can you update the description from "Unlock the drive" to "Password to unlock the drive" or something to indicate that this is the password?

Reply:
or something like "Unlock the drive, specify an optional password here"

Reply:
see how it looks now.